ceed-basis.c

```diff
@@ -252,7 +252,14 @@ static int CeedScalarView(const char *name, const char *fpformat, CeedInt m,
   for (int i=0; i<m; i++) {
     if (m > 1) fprintf(stream, "%12s[%d]:", name, i);
     else fprintf(stream, "%12s:", name);
-    for (int j=0; j<n; j++) fprintf(stream, fpformat, a[i*n+j]);
+    for (int j=0; j<n; j++) {
+      if (fabs(a[i*n+j]) > 1E-14) {
+        fprintf(stream, fpformat, a[i*n+j]);
+      }
+      else {
```

Comment:
`make astyle` would have changed this to `} else {`, but I think it would be cleaner with `? : ` notation. Can you make that small change?

Reply:
Done